e2e/cypress/integration/plugins/upgrade_spec.js

```diff
@@ -99,6 +99,52 @@ describe('Plugin remains enabled when upgraded', () => {
         // * Verify v0.2.0 of plugin
         cy.findByText(/0.2.0/).scrollIntoView().should('be.visible');
     });
+
+    it('MM-T39 Disable Plugin on Removal', () => {
+        // # Install demo plugin and enable it
+        cy.apiAdminLogin();
+        cy.apiInstallPluginFromUrl(
+            'https://github.com/mattermost/mattermost-plugin-demo/releases/download/v0.1.0/com.mattermost.demo-plugin-0.1.0.tar.gz',
```

Comment:
Maybe put this in a constant since it's used in two places?

Reply:
Good suggestion :+1: